framework/src/main/java/org/tron/core/Wallet.java

```diff
@@ -281,16 +285,17 @@ public static String encode58Check(byte[] input) {
   }
 
   public Account getAccount(Account account) {
-    AccountStore accountStore = dbManager.getAccountStore();
+    AccountStore accountStore = chainBaseManager.getAccountStore();
     AccountCapsule accountCapsule = accountStore.get(account.getAddress().toByteArray());
     if (accountCapsule == null) {
       return null;
     }
-    BandwidthProcessor processor = new BandwidthProcessor(dbManager);
+    BandwidthProcessor processor = new BandwidthProcessor(dbManager, chainBaseManager);
```

Comment:
Why pass chain base manger to BandwidthProcessor.  You can get chain base manage from dbmanage.